native/python/pyjp_array.cpp

```diff
@@ -271,7 +271,14 @@ static int PyJPArray_assignSubscript(PyJPArray *self, PyObject *item, PyObject *
 static void PyJPArray_releaseBuffer(PyJPArray *self, Py_buffer *view)
 {
 	JP_PY_TRY("PyJPArrayPrimitive_releaseBuffer");
-	JPContext *context = PyJPModule_getContext();
+	JPContext* context = JPContext_global;
+	if (!context->isRunning())
+	{
+		delete self->m_View;
+		self->m_View = NULL;
+		return;
+	}
+	//	JPContext *context = PyJPModule_getContext();
```

Comment:
This resolves the issue, but leaves one hole.   If there is a memory buffer left open like this and the JVM is gone then accessing the memory will segfault.   I looked through everything I could finding in the Python docs and it looks like the never considered the case of a memory buffer that they can't hold open with reference counting.

Reply:
Mhm, who allocated this buffer in first place?

Reply:
Allocated pyjp_array.cpp at line 314 and line 361 on getBuffer methods.